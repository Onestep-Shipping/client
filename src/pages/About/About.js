import './About.css';

import Header from '../../components/Header/Header.js';
import React from 'react';

const About = () => {
  return (
    <div className="homepage-container">
      <Header />
      <div className="body-container2">
        About
      </div>
    </div>
  );
};

export default About;
