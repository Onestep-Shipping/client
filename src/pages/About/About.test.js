import About from './About';
import React from 'react';
import { shallow } from 'enzyme';

let wrapper = shallow(<About />);

describe('About', () => {
  it('should render the About Component correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});