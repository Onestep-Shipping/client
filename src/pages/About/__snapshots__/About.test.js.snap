// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render the About Component correctly 1`] = `
<div
  className="homepage-container"
>
  <Header />
  <div
    className="body-container2"
  >
    About
  </div>
</div>
`;
